src/Script.js

```diff
@@ -243,6 +254,13 @@ $(document).keydown(function(e) {
 	}
 });
 
+function beautifyScript() {
+    var editor = ace.edit('javascript-editor');
+    var code = editor.getValue();
+    code = beautify(code, { indent_size: 4 })
```

Comment:
4? 4??? How big of a screen do you think I'm playing this game on! And what if I wanted it to be tabs instead of spaces??? 😀 

Joking aside, this looks good. In the future we might want to add more options to the editor though.

Reply:
ARE WE GONNA HAVE A TAB OVER SPACE DEBATE !!!11111!!!11!!111!!!1!11!!11!11!!111!!!1oneone11111!1!!!1!!!

I'm all for more options, I just felt like we should push *something* first and then customize later.